lib/order.rb

```diff
@@ -0,0 +1,30 @@
+require_relative 'checkout'
+
+class Order
+  attr_accessor :order_list, :menu_list
+
+  include Checkout
+
+  def initialize
+  	@menu_list = {"Bottle of Water" => 3.08, "Disney Themed Napkins (x5)" => 50, "Carbonara" => 0.4, "Ice cream" => 0.48, "Happiness" => 0}
+  	@order_list = []
+  end
+
+  def print_menu
+  	menu_list.each do |dish_key, dish_price|
+      "#{dish_key}: £#{dish_price}"
```

Comment:
Not sure what this method is supposed to achieve... Remember that the return value of each is merely the array that it is called on. Should you have used #map?